index.html

```diff
@@ -1,111 +1,579 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <!-- <link rel="stylesheet" href="style.css"> --> <!-- uncomment styles --> 
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-</head>
-<body>
-    <div id="container" class="container">
-        <div class="profile-folder">
-          <div class="folder">
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bakbak+One&family=Poppins:wght@400;500;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="all.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home page</title>
+  </head>
+  <body>
+    <header>
+      <a class="logo" href="#">Binabox</a>
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="#" class="nav-link">Home</a>
+          </li>
+          <li>
+            <a href="blog.html" class="nav-link">Blog</a>
+          </li>
+          <li>
+            <a href="catalog.html" class="nav-link">Catalog</a>
+          </li>
+          <li>
+            <a href="contact-us.html" class="nav-link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div>
+        <input type="text" size="2" />
+        <a href="link" class="button-discord" target="blank">Discord</a>
+        <button class="button-blue" type="button">Connect</button>
+      </div>
+    </header>
+    <main>
+      <section class="statistic">
+        <h1 class="statistic-baner">WITH BINABOX EXPLORE NFT COLLECTION</h1>
+        <p class="statistic-title">
+          We are tne best way to check the rarity on NTF collection
+        </p>
+        <dl class="definition">
+          <dt class="definition-term">2240+</dt>
+          <dd class="definition-description">Total item</dd>
+          <dt class="definition-term">1000+</dt>
+          <dd class="definition-description">Profiles whitelisted</dd>
+        </dl>
+        <button type="button">Connect walet</button>
+        <button type="submit">whitelist now</button>
```

Comment:
It's better to use `type="button"` here, since we don't have a form to submit data